src/animations/backwardsCompatibleWrapper.js

```diff
@@ -0,0 +1,101 @@
+import {
+  always,
+  block,
+  call,
+  clockRunning,
+  cond,
+  set,
+  startClock,
+  stopClock,
+} from '../base';
+import { default as Clock } from '../core/AnimatedClock';
+import { default as Value } from '../core/AnimatedValue';
+
+export default function backwardsCompatibleWrapper(node, AnimationClass) {
+  return (clock, state, config) => {
```

Comment:
as this arrow method seems pretty long it would be better to extract it to the top level and use `function` to call it somehow, then use if the below condition fails

This will also give us an opportunity to free up variable names for clock etc